src/components/Main/Main.jsx

```diff
@@ -0,0 +1,82 @@
+// Import methods from librairies
+import { useContext } from "react";
+
+// Import styles
+import "./Main.css";
+
+// Import components
+import ScheduleCard from "../ScheduleCard/ScheduleCard";
+import RecipeCard from "../RecipeCard/RecipeCard";
+
+//Import context
+import { CurrentRecipesContext } from "../../contexts/currentRecipesContext";
+
+function Main({
+  onRecipeCardClick,
+  onShowMoreClick,
+  onScheduleClick,
+  onDeleteClick,
+  onResetSearch,
+  displayedCards,
+  schedule,
+  isShowMore,
+}) {
+  const recipesList = useContext(CurrentRecipesContext);
+  return (
+    <main className="main">
+      <div className="main__schedule">
+        {schedule.map((item) => {
+          return (
+            <ScheduleCard
+              key={item.dayIndex}
+              day={item.name}
+              dayIndex={item.dayIndex}
+              recipesOfDay={item.scheduledRecipes}
+              onScheduleClick={onScheduleClick}
+              onCardClick={onRecipeCardClick}
+              onDeleteClick={onDeleteClick}
+            />
+          );
+        })}
+      </div>
+      <h2 className="main__container-title">WorldWide Recipes </h2>
+
+      <div className="main__recipe-container">
+        {displayedCards.map((recipe) => {
+          return (
+            <RecipeCard
+              key={recipe._id}
```

Comment:
`              key={recipe._id}
`

Good job that you've used `_id` for the `key` rather than the `index`.

Here is why we shouldn’t use `index` for that: [click here]( https://programmingwithmosh.com/react/why-do-i-need-keys-in-react-lists/#:~:text=React%20recommends%20that%20you%20do,to%20some%20unstable%20component%20behaviour.)